pages/contribute-to-the-protocol.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Contribute to the Protocol
+keywords: governance, security program, Bitmark Upgrade Proposal
+last_updated: 
+sidebar: mydoc_sidebar
+permalink: /contribute-to-the-protocol
+folder: 
+---
+
+
+# Contribute to the Protocol
+_Last updated: 24-FEB 2020_
+
+[TOC]
+
+
+## About the Bitmark Protocol
+
+We expect you probably got here from https://bitmark.com.  But just in case you didn’t, the explanation of the Bitmark Protocol is here: 
+
```

Comment:
People might get here from searches. It's never good to presume how someone got to a web page. I'd suggest cutting this first paragraph (above the links), then sticking those links a little further into this section.